justfile

```diff
@@ -11,8 +11,27 @@ tag := env('CHARMLIBS_TAG', '')
 _uv_run_with_test_requirements := 'uv run --with-requirements ' + quote(join(justfile_dir(), 'test-requirements.txt')) + ' --python ' + python
 
 # this is the first recipe in the file, so it will run if just is called without a recipe
+_short_help:
+    @echo '{{BOLD}}List all commands with {{CYAN}}`just help`{{NORMAL}}{{BOLD}}, or:{{NORMAL}}'
+    @echo '🛠️  Create a new package: {{CYAN}}`just init`{{NORMAL}} or {{CYAN}}`just interface init`{{NORMAL}}'
```

Comment:
I find all of these emojis just add noise. Icons are good shorthands for things you're going to be seeing over and over again, or for obvious things like pass (green check) and fail (red cross). But here IMO they're not adding anything, and distract from the actual content. I'd just drop them, or perhaps replace with `-`.